src/serde/fields.py

```diff
@@ -852,6 +853,44 @@ def __init__(self, **kwargs):  # noqa: N807
 del create_primitive
 
 
+# A helper function...
+def round_decimal(
+    decimal_obj: decimal.Decimal, num_of_places: int = 6
+) -> decimal.Decimal:
+    return decimal_obj.quantize(decimal.Decimal(10) ** -num_of_places).normalize()
+
+
+class Decimal(Instance):
+    """
+    A `~decimal.Decimal` field.
+
+    This field serializes `~decimal.Decimal` objects as strings and
+    deserializes string representations of Decimals as `~decimal.Decimal`
+    objects.
+
+    The resolution of the decimal can be specified. It defaults to 6 decimal places.
+
+    Args:
+        resolution (int): The amount of decimal places to round to and deserialize to.
+        **kwargs: keyword arguments for the `Field` constructor.
+    """
+
+    ty = decimal.Decimal
+
+    def __init__(self, resolution=6, **kwargs):
```

Comment:
The rounding is a hang-over from another time I used decimals, I used it because when float objects are converted to decimal, they retain the slight inaccuracy from having been a float. e.g.

```py
example_float = 0.1
print(Decimal(example_float))

# > 0.10000000000000000000000342858743680232

# Something like this occurs, which is likely to throw up confusion
```

The rounding is simply a personal preference, and it's probably better for people to learn that lesson.

I'll put a note about it in the docstring tomorrow.

Reply:
I agree the default should be without rounding...